pkg/autodiscovery/listeners/dbm_aurora.go

```diff
@@ -0,0 +1,252 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2020-present Datadog, Inc.
+
+package listeners
+
+import (
+	"context"
+	"fmt"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/databasemonitoring/aws"
+	dbmconfig "github.com/DataDog/datadog-agent/pkg/databasemonitoring/config"
+	"github.com/DataDog/datadog-agent/pkg/util/containers"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"strconv"
+	"sync"
+	"time"
+)
+
+const dbmAdIdentifier = "database_monitoring_aurora"
+
+func init() {
+	Register(dbmAdIdentifier, NewDBMAuroraListener)
+}
+
+// DBMAuroraListener implements database-monitoring aurora discovery
+type DBMAuroraListener struct {
+	sync.RWMutex
+	newService       chan<- Service
+	delService       chan<- Service
+	stop             chan bool
+	services         map[string]Service
+	config           dbmconfig.AutodiscoverClustersConfig
+	awsClients       map[string]aws.RDSClient
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏`awsRdsClients`